### DISCUSSION TITLE
INITializing Diskettes

### DISCUSSION THREAD

**Discussion by pfcushing** [thehighnibble/thehighnibble.github.io#49] [Cromemco Z-1 replica]
I just recently started playing around with my IMSAI 8080 booted up as a Cromemco Z-1 running Build 1.11.0.  I find that I am unable to INITialize any diskettes in any of the floppy drives.  I  mount a diskette into the B (or  C) drive and enter the command INIT271.  The program loads and I answer the questions presented and the computer thinks about it for a few seconds and returns a "Drive B Not Ready".  At this point I have to do a restart and the diskette I was trying to INIT disappears from the drive and LIB folder.

Any ideas what I am doing wrong?

**Comment by riwide** [comment]
I booted CDOS258-8 in floppy A, mounted a floppy in B: and used init271 successfully.

When started Cromemco flavour, did you set switches and to a config to NVR (I used switches 0000 1001 0100 0000).
Check that you can mount and unmount a floppy without problems - I did have a state where it wouldn't let me, which I think was because I hadn't done the proper init sequence.

**Comment by thisspaceleftblank** [comment]
My problems, similiar, were corrected by changing the nvram settings.

**Reply by pfcushing** [reply]
yup, those are what I am using for CDOS258 and CPM2.2 but I restarted it again, verified I can mount and unmount a disk into all four drives but still gets Drive Not Ready when I try to run Init271.  It does try at least 'cause i get a "Cylinder" message before it quits.  But No Go.

**Reply by udo-munk** [reply]
Could be a corrupt disk image. Copy a good disk image that is working to the one you want to format and try again.
